packages/hoppscotch-backend/src/admin/admin.model.ts

```diff
@@ -1,4 +1,5 @@
-import { ObjectType } from '@nestjs/graphql';
+import { ObjectType, PartialType } from '@nestjs/graphql';
+import { User } from 'src/user/user.model';
 
 @ObjectType()
-export class Admin {}
+export class Admin extends PartialType(User) {}
```

Comment:
Using the `PartialType` here is technically incorrect right, because we want the `Admin` type to inherit all the properties of the `User` object. From the image below we see that some of the properties from the `User` type arent listed in the `Admin` type at all.

<img width="268" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/14091627/84fcf2c8-3d26-45ea-9f69-97c87adf6a5b">